### PR TITLE
solved exercise 1 - Roelof Wobben

### PR DESCRIPTION
**Pull request by roelof1967**
Here my solution.

### CODE REVIEW

File: exercise_1.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+var Hapi = require('hapi');
+var package = require('./package.json')
+
+var server = new Hapi.Server();
+
+server.connection({ 
+  port: 3000
```
Comment (hueniverse) [inline exercise_1.js]: Assignment asked for port 8000.

File: exercise_1.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+var Hapi = require('hapi');
```
Comment (hueniverse) [inline exercise_1.js]: Take a look at the other pull requests in terms of file names and locations. This should be in `lib/index.js`.

**Comment by roelof1967** [comment]
I have changed this but by a error of me the changes are push in a seperate one. I apolize for this.

**Comment by AdriVanHoudt** [comment]
Reference: https://github.com/hueniverse/hueniversity/pull/27

@roelof1967 you can close this PR if you want to continue with the other one

**Pull request closed without merging**